### PR TITLE
Update CODEOWNERS

### LINKED ISSUE #290: Update CODEOWNERS to replace TEF representative
Hi all,

It has been a pleasure to be part of this initiative where I have learned a lot of things and met amazing people from telcos and companies from all over the world, now I am taking on a new challenge in my career and my colleague @JoseMConde will lead the meetings, so he must be added in the CODEOWNERS file.

Thank you very much, all the best!
Javier Loza

### PR DESCRIPTION
**Pull request by JoseMConde**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:
Update CODEOWNERS

#### Which issue(s) this PR fixes:

Fixes #290

### CODE REVIEW

**Review by eharrison24 — APPROVED** [review]

**Comment by hdamker** [comment]
> Should Maintainers.md also be updated?

@gainsley not necessarily, @javierlozallu was not part of the Maintainer team.

PR LGTM and @camaraproject/simple-edge-discovery_codeowners team already updated. Just waiting for an approval from the EdgeCloud team, then we are good to go.

**Review by gainsley — APPROVED** [review]

**Review by hdamker — APPROVED** [review]

**Pull request merged**
